test/t8n/t8n.cpp

```diff
@@ -47,7 +47,7 @@ int main(int argc, const char* argv[])
 
         if (arg == "-v")
         {
-            std::cout << "evmone dev\n";  // FIXME: Return proper version.
+            std::cout << "evmone " << PROJECT_VERSION << std::endl;
```

Comment:
```suggestion
            std::cout << "evmone-t8n " << PROJECT_VERSION << std::endl;
```

Why not?

Reply:
Right.

Reply:
Depends what you want to see in the generated tests.

Reply:
I think -t8n is ok. We will probably will have also project versions for other executables so it's good to distinct them